### PR TITLE
Feature/navigator

### PR DESCRIPTION
**Pull request by ConnorStroomberg**
Navigator plugin

Using vue for frond-end, flow type hinting, sinon for mocking api requests when testing the store
molgenis-api is loaded npm package

### CODE REVIEW

**Comment by ConnorStroomberg** [comment]
test this please

File: navigator/pom.xml [hunk 1]
```diff
@@ -0,0 +1,107 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>molgenis-apps-core</artifactId>
+        <groupId>org.molgenis</groupId>
+        <version>1.0.0-SNAPSHOT</version>
```
Comment (bartcharbon) [inline navigator/pom.xml]: We need something like "1.0.0-APPS-SNAPSHOT" to distinguish between the 1.0.0 release of the molgenis/molgenis repo and this new apps repo

Comment (ConnorStroomberg) [inline navigator/pom.xml]: Can we not use the artifactId and groupId for this ? molgenis/molgenis is 'just' a dependancy in this setup.

File: navigator/src/main/frontend/src/components/Navigator.vue [hunk 2]
```diff
@@ -43,7 +43,10 @@
     <b-table bordered :items="items" :fields="fields" :filter="filter" class="text-left">
       <template slot="label" scope="label" >
         <span v-if="label.item.type === 'entity'">
-            <a :href="'/menu/main/dataexplorer?entity=' + label.item.id" target="_blank">
+            <span v-if="label.item.abstract">
+              {{label.item.label}} (abstract)
```
Comment (bartcharbon) [inline navigator/src/main/frontend/src/components/Navigator.vue]: Lets ask @mswertz, but I think abstract entities should not be shown at all, the goal of this plugin is to navigate your actual data.

**Pull request merged**